src/views/List.jsx

```diff
@@ -7,6 +8,10 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
+				{React.Children.toArray(
```

Comment:
Why did you choose `React.Children.toArray` over rendering a list of JSX element using `data.map`? 

You could map over the array items and return JSX like you already have inside React.Children.toArray() and it should also work.